presentations/2024-01-25_coffee-and-coding/index.qmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Coffee and Coding"
+subtitle: "{targets}"
+author: "[Jacqueline Grout](mailto:jacqueline.grout1@nhs.net) and [Matt Dray](mailto:matt.dray@nhs.net)"
+date: 2024-01-25
+date-format: "MMM D, YYYY"
+knitr:
+  opts_chunk: 
+    eval: false
+    echo: true
+format:
+  revealjs:
```

Comment:
If you want, you can set incremental reveal to be the default on every slide, which will prevent the need for `::: incremental` blocks on each slide. (If you make this change, you can _stop_ incremental reveal on a slide with `::: nonincremental` block.)
```suggestion
  revealjs:
    incremental: true
```